### PR TITLE
Rich links

### PR DESCRIPTION
**Pull request by regiskuckaertz**
Started working on styling rich links. Still some work to do but wanted to put it out there so you can have a look.

### CODE REVIEW

**Review by Joshua-David — APPROVED** [review]
Looks good 👍

**Comment by regiskuckaertz** [comment]
Ok I'm now feature complete:
- opinion / sport features
- media
- live blogs
- avatars

File: onward/app/views/fragments/richLinkBody.scala.html [hunk 1]
```diff
@@ -42,10 +43,18 @@
...
-                    @fragments.inlineSvg("quote", "icon")
+                @if(!experiments.ActiveExperiments.isParticipating(experiments.Garnett)) {
+                    @if(content.content.cardStyle == Comment ||
+                        (content.content.cardStyle == SpecialReport && content.content.hasTonalHeaderByline)
+                    ) {
+                        @fragments.inlineSvg("quote", "icon")
+                    }
+                } else {
+                    @if(content.content.cardStyle == Comment ||
+                        (content.content.cardStyle == Feature && content.content.metadata.pillar.exists(_.toString.toLowerCase == "sport"))
```
Comment (regiskuckaertz) [inline onward/app/views/fragments/richLinkBody.scala.html]: Don't know if there's a better way to get the pillar name? (I would've thought `Pillar` formed an ADT but apparently not)

**Review by Joshua-David — APPROVED** [review]

File: static/src/stylesheets/module/onward-garnett/_right-hand-most-popular.scss [hunk 2]
```diff
@@ -4,10 +4,6 @@
 
 .right-most-popular__heading {
     border-top: 0;
-
-    &:before {
```
Comment (regiskuckaertz) [inline static/src/stylesheets/module/onward-garnett/_right-hand-most-popular.scss]: Haven't been able to actually see this one work

Comment (regiskuckaertz) [reply static/src/stylesheets/module/onward-garnett/_right-hand-most-popular.scss]: Actually I just have, so restoring.

**Pull request merged**